docs/sphinx/source/installation.rst

```diff
@@ -154,7 +154,7 @@ referred to as *conda environments*, but they're the same for our purposes.
    ``conda create --name pvlibdev python pandas scipy``
 #. **Activate** the new conda environment: ``source activate pvlibdev``
 #. **Install** additional packages into your development environment:
-   ``conda install jupyter ipython matplotlib seaborn nose flake8``
+   ``conda install jupyter ipython matplotlib seaborn pytest flake8``
```

Comment:
We actually currently need both nose and pytest due to our use of a pandas test function that requires nose.